ironfish/src/webApi.ts

```diff
@@ -160,7 +160,8 @@ export class WebApi {
           id: mint.asset.id().toString('hex'),
           metadata: BufferUtils.toHuman(mint.asset.metadata()),
           name: BufferUtils.toHuman(mint.asset.name()),
-          owner: mint.asset.owner().toString('hex'),
+          // TODO(mat): Rename this when the API is updated; will need to be released together
```

Comment:
Nit: consider linking an issue or a document so that future readers can have more context on the meaning of this